### DISCUSSION TITLE
Add SSO for v3.x

### DISCUSSION THREAD

**Discussion by soymgomez** [cachethq/cachet#4426] [General]
Hello,

I wanted to make a PR to add the possibility of SSO in the v3.x. I understand that I should do it in the cachethq/cachet project in the 3.x branch since that is where the user model exists.

If you see this well I see to make a PR to be able to review it.

Best regards

**Comment by jbrooksuk** [comment]
This is awesome, thanks for your interest in contributing to Cachet! I'd actually build this into the `core` package and use the `cachet.user_model` configuration value to find the User model.
